app/Http/Controllers/User/CommentController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers\User;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\User;
+use App\Blog;
+use App\Comments;
+use Auth;
+
+class CommentController extends Controller
+{
+    /**
+     * Handle get list comment.
+     *
+     * @param Request $request
+     * @return array $listComments
+     */
+    public function index(Request $request, $blogId)
+    {
+        return $listComments = $this->getCommentsbyBlogId($blogId);
+    }
+
+    /**
+     * Handle add new comment and load new list comment.
+     *
+     * @param Request $request
+     * @return array
+     */
+    public function create(Request $request)
+    {
+        $user = Auth::user();
+        try {
+            $comment =  new Comments();
+
+            $comment->user_id = $user->id;
+            $comment->blog_id = $request['blogId'];
+            $comment->comment_content = $request->comment_content;
+
+            $comment->save();
+        } catch (Exception $e) {
+            return [
+                'error' => 1,
+                'message' => $e->getMessage()
+            ];
+        }
+        $listComments = $this->getCommentsbyBlogId($request['blogId']);
```

Comment:
why need return listComments after create ?

Reply:
After create, I'll reload new list comment. This will has new comment of user.

Reply:
1 function only do 1 work , please use other way to do your reload

Reply:
I fixed